File: PERL/Cap05/c05ex09.pl
# c05ex09.pl

#!/usr/bin/perl

use strict;
use warnings;

my $I = 1;
inicio:
  printf "Entre um valor numerico: ";
  my $N = <STDIN>;
  chomp $N;

  my $R = $N * 3;

  print "Resultado = $R\n";
  print "\n";

  if ($I > 4) {goto fim}
  $I = $I + 1;
  
goto inicio;
fim:

print "\n";
print "Tecle <Enter> para encerrar... ";
<STDIN>;

File: GO/Cap05/c05ex09.go
// c05ex09.go

package main

import "fmt"

func main() {
  var I, N, R int

  for I = 1; I <= 5; I++ {
    fmt.Print("Entre um valor numerico: ")
    fmt.Scan(&N)

    R = N * 3

    fmt.Println("Resultado =", R)
    fmt.Println()
  }

  fmt.Println()
  fmt.Print("Tecle <Enter> para encerrar... ")
  fmt.Scanln()
}
